src/kernel/checks.cpp

```diff
@@ -7,21 +7,22 @@
 #include <key.h>
 #include <random.h>
 #include <util/time.h>
+#include <util/translation.h>
```

Comment:
nit in the last commit. Maybe have this file checked by iwyu?

https://github.com/bitcoin/bitcoin/pull/25065#discussion_r890170373

Reply:
re: https://github.com/bitcoin/bitcoin/pull/25308#discussion_r892505713

> nit in the last commit. Maybe have this file checked by iwyu?

Sure, done in new commit